internal/clients/diffreviewer/circleci/circleci_service.go

```diff
@@ -0,0 +1,106 @@
+package circleci
+
+import (
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/nightfallai/nightfall_code_scanner/internal/clients/diffreviewer"
+	"github.com/nightfallai/nightfall_code_scanner/internal/clients/gitdiff"
+	"github.com/nightfallai/nightfall_code_scanner/internal/clients/logger"
+	circlelogger "github.com/nightfallai/nightfall_code_scanner/internal/clients/logger/circle_logger"
+	"github.com/nightfallai/nightfall_code_scanner/internal/clients/nightfall"
+	"github.com/nightfallai/nightfall_code_scanner/internal/interfaces/gitdiffintf"
+	"github.com/nightfallai/nightfall_code_scanner/internal/nightfallconfig"
+)
+
+const (
+	WorkspacePathEnvVar        = "GITHUB_WORKSPACE"
+	NightfallAPIKeyEnvVar      = "NIGHTFALL_API_KEY"
+	CircleRepoNameEnvVar       = "CIRCLE_PROJECT_REPONAME"
```

Comment:
may or may not need some of these env vars based on what we need for the github Issues API